k8s/conf/nsm-configmap.yaml

```diff
@@ -0,0 +1,7 @@
+---
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: nsm-config
+data:
+  excluded_prefixes.yaml: ''
```

Comment:
Is this documented somewhere?

Reply:
I guess no, where do think it should be documented?

Reply:
`docs/spec` maybe?

Reply:
thank you!
- [ ] create documentation for nsm-configmap in docs/spec